scripts/validate-packages.py

```diff
@@ -3,13 +3,16 @@
 import json
 import jsonschema
 import os
+import re
 import sys
 from distutils.version import LooseVersion
 
 SCRIPTS_DIR = os.path.dirname(os.path.realpath(__file__))
 UNIVERSE_DIR = os.path.join(SCRIPTS_DIR, "..")
 PKG_DIR = os.path.join(UNIVERSE_DIR, "repo/packages")
 SCHEMA_DIR = os.path.join(UNIVERSE_DIR, "repo/meta/schema")
+LETTER_PATTERN = re.compile("^[A-Z]$")
+PACKAGE_FOLDER_PATTERN = re.compile("^[a-z][a-z0-9-]+[a-z0-9]$")
```

Comment:
@takirala regarding your question: yes, let's change the `+` to a `*`. There might be a package name that's only two characters long; consider the S3 object store or the H2 in-memory database.